modules/terraform/cmd.go

```diff
@@ -49,8 +52,20 @@ func GetCommonOptions(options *Options, args ...string) (*Options, []string) {
 		options.TerraformBinary = DefaultExecutable
 	}
 
-	if options.TerraformBinary == "terragrunt" {
+	if options.TerraformBinary == TerragruntDefaultPath {
 		args = append(args, "--terragrunt-non-interactive")
+		// for newer Terragrunt version, setting simplified log formatting
+		if options.EnvVars == nil {
+			options.EnvVars = map[string]string{}
+		}
+		_, tgLogSet := options.EnvVars["TERRAGRUNT_LOG_FORMAT"]
+		if !tgLogSet {
+			options.EnvVars["TERRAGRUNT_LOG_FORMAT"] = "key-value"
```

Comment:
Can you explain what this does? It could be great to add some comments explanining the default behaviour here and below.

Reply:
Added description